certif/app/controllers/authenticated/team/invite.js

```diff
@@ -3,13 +3,38 @@ import { service } from '@ember/service';
 import { action } from '@ember/object';
 
 export default class AuthenticatedTeamInviteController extends Controller {
+  @service currentUser;
+  @service intl;
+  @service notifications;
   @service router;
+  @service store;
 
   email = null;
 
   @action
   async createCertificationCenterInvitation(event) {
     event.preventDefault();
+
+    const certificationCenterId = this.currentUser.currentAllowedCertificationCenterAccess.id;
+    const emailsTextareaInput = Object.values(event.target)
+      .filter((target) => target.type === 'textarea')
+      .shift();
```

Comment:
Ca permet de gerer quel cas concret?

Reply:
Je viens de retirer le code car pas nécessaire. On a déjà `this.emails` 😅